README.md

```diff
@@ -63,3 +63,41 @@ Line what is shown up will request all metricdata, if only some of metric data i
 ```
 Names should be seperated with ',' and there should not be any spaces.
 You can find the different monitoring data metric names from the document: [X-Road EnvironmentalMonitoring](https://github.com/vrk-kpa/X-Road/tree/develop/doc/EnvironmentalMonitoring)
+
+## SSL
+
+To enable secure HTTPS connection to central monitoring client security server with mutual authentication follow the steps below.
+
+Create new keystore and keypair for xroad-monitor-collector
+```
+keytool -alias xroad-monitor-collector -genkeypair -keystore /etc/xroad/xroad-monitor-collector/keystore -validity 7300 -keyalg RSA -keysize 2048 -sigalg SHA256withRSA -dname C=FI,CN=xroad-monitor-collector
+```
+
+Export the xroad-monitor-collector certificate to file
+```
+keytool -keystore /etc/xroad/xroad-monitor-collector/keystore -exportcert -rfc -alias xroad-monitor-collector > xroad-monitor-collector.cer
+```
+
+Using the security server administrator interface configure the security server to use HTTPS connection and import the xroad-monitor-collector certificate from previous step to "Internal servers "- "Internal TLS certificates" list.
+
+Download the security server's certificate with openssl
```

Comment:
Instead of parsing openssl output, one could also export the security server internal certificate using the admin interface (system parameters->internal tls certficate->export).